src/screens/ChannelListScreen/elements/components/PostNotificationPreview.js

```diff
@@ -49,7 +49,7 @@ const styles = StyleSheet.create({
     maxWidth: '70%'
   },
   subtitleStyle: {
-    // color: COLORS.blackgrey,
+    // color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out color property `// color: COLORS.gray400,` should be removed if it's not being used to keep the code clean.
```diff
-    // color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->